live-examples/css-examples/filter-effects/filter.html

```diff
@@ -0,0 +1,49 @@
+<section id="example-choice-list" class="example-choice-list large" data-property="filter">
+  <div class="example-choice">
+    <pre><code class="language-css">filter: url("../../media/examples/shadow.svg#element-id");</code></pre>
+    <button type="button" class="copy hidden" aria-hidden="true">
+      <span class="visually-hidden">Скопіювати в буфер</span>
+    </button>
+  </div>
+
+  <div class="example-choice">
+    <pre><code class="language-css">filter: blur(5px);</code></pre>
+    <button type="button" class="copy hidden" aria-hidden="true">
+      <span class="visually-hidden">Скопіювати в буфер</span>
+    </button>
+  </div>
+
+  <div class="example-choice">
+    <pre><code class="language-css">filter: contrast(200%);</code></pre>
+    <button type="button" class="copy hidden" aria-hidden="true">
+      <span class="visually-hidden">Скопіювати в буфер</span>
+    </button>
+  </div>
+
+  <div class="example-choice">
+    <pre><code class="language-css">filter: grayscale(80%);</code></pre>
+    <button type="button" class="copy hidden" aria-hidden="true">
+      <span class="visually-hidden">Скопіювати в буфер</span>
+    </button>
+  </div>
+
+  <div class="example-choice" initial-choice="true">
+    <pre><code class="language-css">filter: hue-rotate(90deg);</code></pre>
+    <button type="button" class="copy hidden" aria-hidden="true">
+      <span class="visually-hidden">Скопіювати в буфер</span>
+    </button>
+  </div>
+
+  <div class="example-choice">
+    <pre><code class="language-css">filter: drop-shadow(16px 16px 20px red) invert(75%);</code></pre>
+    <button type="button" class="copy hidden" aria-hidden="true">
+      <span class="visually-hidden">Скопіювати в буфер обміну</span>
```

Comment:
```suggestion
      <span class="visually-hidden">Скопіювати в буфер</span>
```
Нехай це буде однаковий текст хоча б у межах одного прикладу